app/services/check_course_jobs.rb

```diff
@@ -6,9 +6,27 @@
 class CheckCourseJobs
   include CourseQueueSorting
 
+  def self.remove_orphan_locks(courses_to_update)
+    orphan_lock_courses = []
+    courses_to_update.each do |course|
+      check_course_job = new(course)
+      is_removed = check_course_job.delete_orphan_lock
+      orphan_lock_courses << check_course_job.sentry_extra if is_removed
+    end
+
+    orphan_lock_count = orphan_lock_courses.length
+    return unless orphan_lock_count.positive?
+
+    Raven.capture_message("#{orphan_lock_count} Orphan lock(s) removed",
```

Comment:
This is fine, but I would also like to have logging on the individaul course record — probably using the same data structure as we already do for logging the completion time of each successful update. So, an orphan lock should be treated like an update that happened but failed, and it should increment count of total updates for a course accordingly.

Reply:
Done!